API.md

```diff
@@ -10,15 +10,20 @@ const Log = require('ipfs-log')
 
 ### Constructor
 
-#### new Log(ipfs, [id])
+#### new Log(ipfs, identity, [logId, access])
 
 Create a log. Each log gets a unique ID passed as an argument. Returns a `Log` instance.
 
 ```javascript
-const log = new Log(ipfs, 'logid')
+const IdentityProvider = require('orbit-db-identity-provider')
+const identity = await IdentityProvider.createIdentity({ id: 'peerid' })
+const { AccessController } = Log
+const access = new AccessController()
+const ipfs = new IPFS()
+const log = new Log(ipfs, identity, { logId: 'logid', access: access })
 ```
 
-`ipfs` is an instance of IPFS. `id` is a unique log identifier. Usually this should be a user id or similar.
+`ipfs` is an instance of IPFS. `access` is an instance of `AccessController`, which by default allows any one to append to the log. `identity` is an instance of [Identity](https://github.com/orbitdb/orbit-db-identity-provider/blob/master/src/identity.js), used to sign entries. `id` is a unique log identifier. Usually this should be a user id or similar.
```

Comment:
This could be improved by describing the arguments in the order they appear in the function signature.

Also, `id` should prolly be `logId`?